### PR TITLE
feat: Apply data auto refresh

### PR DESCRIPTION
**Pull request by jinoosss**
### What type of PR is this?

- feature

### What this PR does:
- The wallet periodically patches the account's balance.
- Change to use React Query.

### CODE REVIEW

File: packages/adena-extension/src/pages/popup/wallet/manage-token-added/index.tsx [hunk 1]
```diff
@@ -89,7 +87,7 @@ const ManageTokenAddedContainer: React.FC = () => {
       return;
     }
     await addGRC20TokenMetainfo(selectedTokenInfo);
-    await updateTokenBalanceInfos(tokenMetainfos);
+    // await updateTokenBalanceInfos(tokenMetainfos);
```
Comment (dongwon8247) [inline packages/adena-extension/src/pages/popup/wallet/manage-token-added/index.tsx:90]: Leftover?

File: packages/adena-extension/src/pages/popup/wallet/wallet-main/index.tsx [hunk 2]
```diff
@@ -98,13 +96,13 @@ export const WalletMain = (): JSX.Element => {
...
-  useEffect(() => {
-    if (currentAccount) {
-      updateBalanceAmountByAccount(currentAccount);
-    }
-  }, [currentAccount]);
+  // useEffect(() => {
+  //   if (currentAccount) {
+  //     updateBalanceAmountByAccount(currentAccount);
+  //   }
+  // }, [currentAccount]);
```
Comment (dongwon8247) [inline packages/adena-extension/src/pages/popup/wallet/wallet-main/index.tsx:103]: Leftover?

**Review by dongwon8247 — APPROVED** [review]
Balances in Main and Sidebar are updated well, but TokenDetail's balance doesn't seem to be updated. Can you check? Also I've only tested with $GNOT, but please check with GRC20s too.

image

File: packages/adena-extension/src/pages/popup/wallet/manage-token-added/index.tsx (hunk 1 above)
Comment (jinoosss) [reply packages/adena-extension/src/pages/popup/wallet/manage-token-added/index.tsx]: I missed it, I removed the unused code.

**Comment by jinoosss** [comment]
Thanks for pointing this out to me.
I've fixed it to work on the token details page.

https://github.com/onbloc/adena-wallet/assets/112360739/1d8ed888-3c59-4a01-b4e5-92fbb5663b0e

**Pull request merged**